request-management-api/request_api/services/notifications/notificationuser.py

```diff
@@ -130,4 +132,13 @@ def __getgroupmembers(self,groupid):
                 notificationusers.append({"userid":user["username"], "usertype":notificationusertypelabel})
             return notificationusers 
         return []
-        
\ No newline at end of file
+
+    def __getscanningteam(self):
```

Comment:
Please make the necessary changes mentioned in accordance with callstack.

Reply:
@antsand , use __getgroupmembers method.